incubator-tools/test_harness_guide/test_harness_guide.ipynb

```diff
@@ -0,0 +1,1455 @@
+{
```

Comment:
<p>Remove the comments</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Comments are removed in updated notebook</p>